src/OrderImport/Rules/ZalandoColissimo.php

```diff
@@ -79,167 +68,37 @@ public function isApplicable(OrderResource $apiOrder)
-            $logPrefix . $this->l('Saving Colissimo pickup point.', 'ZalandoColissimo'),
-            'Order'
-        );
+        return $this->l('Set the carrier to Colissimo Pickup Point and add necessary data in the colissimo module accordingly.', 'ZalandoColissimo');
+    }
 
-        // Retrieve necessary order data
+    protected function getProductCode(OrderResource $apiOrder)
+    {
         $apiOrderData = $apiOrder->toArray();
-        list($productCode, $colissimoPickupPointId) = explode(':', $apiOrderData['additionalFields']['service_point_id']);
-
-        $shippingAddress = $apiOrder->getShippingAddress();
-
-        // Save/update Colissimo pickup point
-        $pickupPointData = [
-            'colissimo_id' => $colissimoPickupPointId,
-            'company_name' => $apiOrderData['additionalFields']['service_point_name'],
-            'address1' => $shippingAddress['street'],
-            'address2' => $shippingAddress['street2'],
-            'address3' => '',
-            'city' => $shippingAddress['city'],
-            'zipcode' => $shippingAddress['postalCode'],
-            'country' => Tools::strtoupper(Country::getNameById($params['cart']->id_lang, Country::getByIso($shippingAddress['country']))),
-            'iso_country' => $shippingAddress['country'],
-            'product_code' => $productCode,
-            'network' => '',
-        ];
-        $pickupPoint = ColissimoPickupPoint::getPickupPointByIdColissimo($colissimoPickupPointId);
-        $pickupPoint->hydrate(array_map('pSQL', $pickupPointData));
-        $pickupPoint->save();
-
-        ProcessLoggerHandler::logInfo(
-            $logPrefix .
-                sprintf(
-                    $this->l('Linking Colissimo pickup point %s to cart %s.', 'ZalandoColissimo'),
-                    $colissimoPickupPointId,
-                    $params['cart']->id
-                ),
-            'Order'
-        );
-
-        // Save pickup point/cart association
-        ColissimoCartPickupPoint::updateCartPickupPoint(
-            (int) $params['cart']->id,
-            (int) $pickupPoint->id,
-            '' //TODO $mobilePhone; use billing address phone ? Not required, but might as well, the merchant will have to fill it either way.
-        );
+        $service_point_id = explode(':', $apiOrderData['additionalFields']['service_point_id']);
+        if (count($service_point_id) > 1) {
+            return $service_point_id[1];
+        }
 
-        return true;
+        return 'A2P';
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getConditions()
+    protected function getPointId(OrderResource $apiOrder)
     {
-        return $this->l('If the order comes from Zalando and has non-empty "service_point_id" and "service_point_name" additional fields.', 'ZalandoColissimo');
-    }
+        $apiOrderData = $apiOrder->toArray();
+        list($colissimoPickupPointId) = explode(':', $apiOrderData['additionalFields']['service_point_id']);
```

Comment:
We have two cases with zalando : 
- BPR:123456 with service:pointID
- 123456 with only pointID